package.json

```diff
@@ -46,6 +46,18 @@
 					"default": null,
 					"description": "Optional, defaults to searching for \"php\". The path to a PHP 7.0+ executable to use to execute the Psalm server. The PHP 7.0+ installation should preferably include and enable the PHP module `pcntl`. (Modifying requires restart)"
 				},
+				"psalm.phpExecutableArgs": {
+					"type": [
+						"array",
+						"null"
+					],
+					"default": [
+						"-dxdebug.remote_autostart=0",
+						"-dxdebug.remote_enable=0",
+						"-dxdebug_profiler_enable=0"
+					],
+					"description": "Optional (Advanced), default is '-dxdebug.remote_autostart=0 -dxdebug.remote_enable=0 -dxdebug_profiler_enable=0'.  Additional PHP executable CLI arguments to use."
+				},
```

Comment:
This new setting allows the end-user to specify php cli arguments.  I set the default to disable xdebug, because it is not needed for the psalm language server and xdebug can cause a huge hit on performance.  It is safe to specify these arguments even if xdebug is not installed.